app/components/contributor-list/component.ts

```diff
@@ -1,16 +1,17 @@
+import { computed } from '@ember-decorators/object';
+import { service } from '@ember-decorators/service';
 import { A } from '@ember/array';
 import Component from '@ember/component';
-import { computed } from '@ember/object';
-import { inject as service } from '@ember/service';
 
-export default class ContributorList extends Component.extend({
-    tagName: 'span',
+export default class ContributorList extends Component {
+    @service i18n;
 
-    contributors: A([]),
-}) {
-    i18n = service('i18n');
+    tagName = 'span';
```

Comment:
You can use [@tagName](https://ember-decorators.github.io/ember-decorators/docs/function/addon/component/index.js.html#static-function-tagName)